packages/commonwealth/server/scripts/dbNode.ts

```diff
@@ -125,20 +125,26 @@ async function mainProcess(
     );
 
   let query =`
-    SELECT
-      "Chains"."id",
-      "Chains"."substrate_spec",
-      "url",
-      "private_url",
-      "Chains"."address",
-      "Chains"."base",
-      "Chains"."type",
-      "Chains"."network",
-      "Chains"."ce_verbose"
-    FROM "Chains"
-    JOIN "ChainNodes"
-    ON "Chains".chain_node_id = "ChainNodes".id
-    WHERE "Chains"."has_chain_events_listener" = true;
+  SELECT
+    c."id",
+    c."substrate_spec",
+    cn."url",
+    cn."private_url",
+    con.address as "address",
+    c."base",
+    c."type",
+    c."network",
+    c."ce_verbose"
+  FROM "Chains" c 
+    JOIN "ChainNodes" cn
+      ON c.chain_node_id = cn.id
+    LEFT JOIN "CommunityContracts" cc
+      ON cc.chain_id = c.id
+    LEFT JOIN "Contracts" con
+      ON con.id = cc.contract_id
+  WHERE c."has_chain_events_listener" = true
+    AND (con.type IN ('marlin-testnet', 'aave', 'compound') OR
```

Comment:
This type seems very arbitrary?

Reply:
actually, crazy thought: what if we get rid of the where clause altogether? same set of 11 chains get returned.

Reply:
i removed the WHERE clause as I don't think it's necessary

Reply:
never mind...